lua/session_manager/utils.lua

```diff
@@ -58,7 +58,9 @@ function utils.load_session(filename, discard_current)
     vim.api.nvim_buf_delete(current_buffer, { force = true })
 
     utils.is_session = true
+    vim.cmd[[doautocmd User SessionLoadPre]]
```

Comment:
Please, use `vim.api.nvim_command` instead of `vim.cmd` for single commands. It's just faster.
```suggestion
    vim.api.nvim_command('doautocmd User SessionLoadPre')
```